lib/iris/fileformats/grib/__init__.py

```diff
@@ -32,65 +26,49 @@
 import six
 
 import datetime
-import math  #for fmod
+import math  # for fmod
 import warnings
 
-import biggus
 import cartopy.crs as ccrs
 import cf_units
 import gribapi
 import numpy as np
 import numpy.ma as ma
-import scipy.interpolate
 
-from iris._deprecation import warn_deprecated
-from iris.analysis._interpolate_private import Linear1dExtrapolator
+import iris
+from iris._lazy_data import as_lazy_data
 import iris.coord_systems as coord_systems
-from iris.exceptions import TranslationError
-# NOTE: careful here, to avoid circular imports (as iris imports grib)
+from iris.exceptions import TranslationError, NotYetImplementedError, IrisError
 from iris.fileformats.grib import grib_phenom_translation as gptx
 from iris.fileformats.grib import _save_rules
-import iris.fileformats.grib._load_convert
+from iris.fileformats.grib._load_convert import convert as load_convert
 from iris.fileformats.grib.message import GribMessage
-import iris.fileformats.grib.load_rules
-
-# Issue a blanket deprecation for this module.
-warn_deprecated(
-    "The module iris.fileformats.grib is deprecated since v1.10. "
-    "Please install the package 'iris_grib' package instead.")
 
 
 __all__ = ['load_cubes', 'save_grib2', 'load_pairs_from_fields',
-           'save_pairs_from_cube', 'save_messages', 'GribWrapper',
-           'as_messages', 'as_pairs', 'grib_generator', 'reset_load_rules',
-           'hindcast_workaround']
-
-
-#: Set this flag to True to enable support of negative forecast periods
-#: when loading and saving GRIB files.
-#:
-#: .. deprecated:: 1.10
-hindcast_workaround = False
+           'save_pairs_from_cube', 'save_messages']
```

Comment:
What has happened to GribWrapper from this list?  The class is still in the module (although it has been altered), does it not need to be in here still as it has been undeprecated?